Update README.md

matt392code opened the pull request:
Bug #3085 
Updated the README.md with the links:
https://fprime.jpl.nasa.gov/latest/
https://git-scm.com/
https://gcc.gnu.org/
https://clang.llvm.org/
https://pypi.org/project/pip/
https://www.debian.org/
https://www.virtualbox.org/

| | |
|:---|:---|
|**_Related Issue(s)_**|  |
|**_Has Unit Tests (y/n)_**|  |
|**_Documentation Included (y/n)_**|  |

---
## Change Description

A description of the changes contained in the PR.

## Rationale

A rationale for this change. e.g. fixes bug, or most projects need XYZ feature.

## Testing/Review Recommendations

Fill in testing procedures, specific items to focus on for review, or other info to help the team verify these changes are flight-quality.

## Future Work

Note any additional work that will be done relating to this issue.

matt392code committed:
Update README.md

Bug #3085 
Updated the README.md with the links:
https://fprime.jpl.nasa.gov/latest/
https://git-scm.com/
https://gcc.gnu.org/
https://clang.llvm.org/
https://pypi.org/project/pip/
https://www.debian.org/
https://www.virtualbox.org/

zimri-leisher commented on README.md:
```diff
@@ -17,16 +17,16 @@ F´ (F Prime) is a component-driven framework that enables rapid development and
 - Testing tools for testing flight software at the unit and integration levels.
 
-Learn more about [F´ key features](https://nasa.github.io/fprime/features.html).
+Learn more about [F´ key features]([https://(https://fprime.jpl.nasa.gov/latest/)).
```
it seems to me like there's a typo in this line: an extra opening square bracket.

matt392code replied:
Thank you for catching that.
 - Removed extra bracket on hyperlink
 - Updated website to current one: 
**Please Visit the F´ Website:** [https://fprime.jpl.gov/latest](https://fprime.jpl.nasa.gov/latest/)

zimri-leisher commented on README.md:
```diff
@@ -17,16 +17,16 @@ F´ (F Prime) is a component-driven framework that enables rapid development and
-4. Clang or GNU C and C++ compilers (e.g. gcc and g++)
-5. [Python 3.8+](https://www.python.org/downloads/), virtual environments, and PIP
+4. [Clang](https://clang.llvm.org/) or [GNU C and C++ compilers](https://gcc.gnu.org/) (e.g. gcc and g++)
+5. [Python 3.8+](https://www.python.org/downloads/), [virtual environments](https://www.virtualbox.org/), and [PIP](https://pypi.org/project/pip/)
```
i highly doubt this "virtual environment" noun was meant to reference virtual machines like virtualbox. Likely means something like python virtualenv or conda. Also not sure how useful these links are.

matt392code committed:
Update README.md

 - Removed extra bracket on hyperlink
 - Updated website to current one: 
**Please Visit the F´ Website:** [https://fprime.jpl.gov/latest](https://fprime.jpl.nasa.gov/latest/)

LeStarch committed:
Fixing link for virtual environments

LeStarch committed:
Remove double HTTPS

LeStarch committed:
Spurious [

LeStarch reviewed (approved):
Made some fixes to the links, otherwise looks great!

matt392code commented:
> Made some fixes to the links, otherwise looks great!

Thank you!  Greatly appreciate it.
